### PR TITLE
Anirudh edits

### PR DESCRIPTION
**Pull request by dimberman**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.11 Features: set Milestone to 1.11 and Base Branch to release-1.11
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

**Review by chaimhaas — CHANGES_REQUESTED** [review]
Line 35 - end with 'completely independent' - also, capitalize Docker in that last sentence of that line.

**Review by chaimhaas — CHANGES_REQUESTED** [review]
Line 35 - end with 'completely independent' - also, capitalize Docker in that last sentence of that line.

Line 140 and 144 both say Step 2 - make sure to properly re-number.

**Review by chaimhaas — CHANGES_REQUESTED** [review]
Line 35 - end with 'completely independent' - also, capitalize Docker in that last sentence of that line.

Line 140 and 144 both say Step 2 - make sure to properly re-number.

**Review by chaimhaas — CHANGES_REQUESTED** [review]
Line 13 - capitalize Python in 'Python object DAG'

Line 35 - end with 'completely independent' - also, capitalize Docker in that last sentence of that line.

Line 140 and 144 both say Step 2 - make sure to properly re-number.

**Comment by dimberman** [comment]
@chaimhaas @foxish PTAL

**Comment by dimberman** [comment]
@chaimhaas acked your comments

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 1]
```diff
@@ -1,40 +1,38 @@
...
+Since its inception, Airflow's greatest strength has been its flexibility. Airflow offers a wide range of integrations for services ranging from Spark and HBase, to services on Google Cloud Platform (GCP) and Amazon Web Services (AWS) S3. Airflow also offers easy extensibility through its plug-in framework. However, one limitation of the project is that Airflow users are confined to the frameworks and clients that exist on the Airflow worker at the moment of execution. If a user wishes to use a different version of SciPy or test a new deep learning framework, they would need to either launch a new Airflow cluster or risk conflicting with the dependencies of other users' workflows.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: remove `S3`? Or reword to - "services on various cloud providers"

Comment (foxish) [reply content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: "If a user wishes to use a different version of SciPy or test a new deep learning framework, they would need to either launch a new Airflow cluster or risk conflicting with the dependencies of other users' workflows."

This is a sudden jump. Can we add a line talking about how such a user may be using airflow first, and then go into how dependency management is hard?

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
+Since its inception, Airflow's greatest strength has been its flexibility. Airflow offers a wide range of integrations for services ranging from Spark and HBase, to services on Google Cloud Platform (GCP) and Amazon Web Services (AWS) S3. Airflow also offers easy extensibility through its plug-in framework. However, one limitation of the project is that Airflow users are confined to the frameworks and clients that exist on the Airflow worker at the moment of execution. If a user wishes to use a different version of SciPy or test a new deep learning framework, they would need to either launch a new Airflow cluster or risk conflicting with the dependencies of other users' workflows. 
+
+To address this issue, we've utilized Kubernetes to allow users to launch arbitrary Docker containers and configurations. Airflow users can now have full power over their run-time environments, resources, and secrets, basically turning Airflow into an "any job you want" scheduler.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: arbitrary `Kubernetes pods`? Because the docker operator is different?

Comment (foxish) [reply content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: s/scheduler/workflow orchestrator/

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 3]
```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
 
 ### 2: CI/CD via Jenkins -> Docker Image
 
-There are a multitude on articles on [generating Docker files within a Jenkins build](https://getintodevops.com/blog/building-your-first-docker-image-with-jenkins-2-guide-for-developers). It's a good rule of thumb that you should never use a user-generated Docker image in a production build. By reserving release tags to the Jenkins user, you can ensure that malicious or untested code will never be run by your production Airflow instances.
+There are a multitude on articles on [generating Docker files within a Jenkins build](https://getintodevops.com/blog/building-your-first-Docker-image-with-jenkins-2-guide-for-developers). It's a good rule of thumb that you should never use a user-generated Docker image in a production build. By reserving release tags to the Jenkins user, you can ensure that malicious or untested code will never be run by your production Airflow instances.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: Shall we skip the CI/CD specific part in such detail? It doesn't seem full fledged enough (underspecified in terms of how one can react to the webhook etc) and the example is too simplistic . Maybe we just add a small section about how one *could* use it for CI/CD?

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 4]
```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
...
                           get_logs=True,
                           dag=dag
                           )
 ```
-                          
-                          
+
+# Launching a test deployment
+
+Since the Kubernetes Operator is not yet released, we haven't released an official helm chart or operator (however both are currently in progress). However, we are including instructions for a basic deployment below  and are actively looking for foolhardy beta testers to try this new feature. To try this system out please follow these steps:
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: link to helm

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 5]
```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
...
+To modify/add your own DAGs, you can use `kubectl cp` to upload local files into the DAG folder of the Airflow scheduler. Airflow will then read the new DAG and automatically upload it to its system. The following command will upload any local file into the correct directory:
+
+`kubectl cp <local file> <namespace>/<pod>:/root/airflow/dags -c scheduler`
+
+## Step 6: Enjoy!
+
 # So when will I be able to use this?
 
-The Kubernetes Operator and Kubernetes Executor (article to come) have both been merged into the [1.10 release branch of Airflow](https://github.com/apache/incubator-airflow/tree/v1-10-test) (the executor in experimental mode). While this branch is still in the early stages, we hope to see it released for wide release in the next few months.
+ While this branch is still in the early stages, we hope to see it released for wide release in the next few months.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: It's been merged into master right? So s/branch/feature/?

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 6]
```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
...
+
+# Get Involved
+
+This feature is just the beginning of multiple major efforts to improves Apache Airflow integration into Kubernetes. The Kubernetes Operator has been merged into the [1.10 release branch of Airflow](https://github.com/apache/incubator-airflow/tree/v1-10-test) (the executor in experimental mode), along with a fully k8s native scheduler called the Kubernetes Executor (article to come). These features are still in a stage where early adopters/contributers can have a huge influence on the future of these features.
+
+For those interested in joining these efforts, I'd recommend checkint out these steps: 
+
+ * Join the airflow-dev mailing list at dev@airflow.apache.org.
+ * File an issue in [Apache Airflow JIRA](https://issues.apache.org/jira/projects/AIRFLOW/issues/)
+ * Join our SIG-BIG-DATA meetings on Wednesdays at 10am PT.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: s/SIG-BIG-DATA/SIG-BigData/

File: content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md [hunk 7]
```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
...
+
+I'd like to thank the Apache Spark and Kubernetes communities, particularly Grant Nicholas, Ben Goldberg, Anirudh Ramanathan, Fokko Dreisprong, and Bolke de Bruin, for your awesome help on these features as well as our future efforts.
```
Comment (foxish) [inline content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md]: The post starts off with "We..." taking the voice of the community. So, maybe reword to "Special thanks to ..."?

**Comment by dimberman** [comment]
@chaimhaas @foxish PTAL

**Comment by foxish** [comment]
Thanks! LGTM from my end. Let's ship it.
